### ISSUE TITLE
arrows are not displayed on the images

### ISSUE THREAD

**Issue by idomiralin** [lukasz-galka/ngx-gallery#276]
Hello,

I cannot manage to add arrows or to have them visible (and the background-color is not a problem, because the images have different colors and on neither of them are the arrows visible).
Did I do something wrong? Sorry if it isn't a bug.

I tried adding arrowPrevIcon and arrowNextIcon, but the arrows are not visible with or without these properties.

This is the code:

 `         <ngx-gallery [options]="galleryOptions" [images]="galleryImages"></ngx-gallery>`

`            this.galleryOptions = [
            {
               thumbnails: false,
               preview: false,
               imageBullets: true, 
               width: '300px',
               height: '200px',
               arrowPrevIcon: "fa fa-arrow-circle-o-left", 
               arrowNextIcon: "fa fa-arrow-circle-o-right"
           },
           //max-width 800
           {
            breakpoint: 800,
            width: '200px',
            height: '170px'
           },
           //max-width
           {
             breakpoint: 400
           }
          ]
    
        this.galleryImages = [
          {
            small: 'assets/images/calendar/autumn.jpg',
            medium: 'assets/images/calendar/autumn.jpg',
            big: 'assets/images/calendar/autumn.jpg'
          },
          {
            small: 'assets/images/calendar/spring.jpg',
            medium: 'assets/images/calendar/spring.jpg',
            big: 'assets/images/calendar/spring.jpg'
          },
          {
            small: 'assets/images/calendar/summer.jpg',
            medium: 'assets/images/calendar/summer.jpg',
            big: 'assets/images/calendar/summer.jpg'
          },
          {
            small: 'assets/images/calendar/winter.jpg',
            medium: 'assets/images/calendar/winter.jpg',
            big: 'assets/images/calendar/winter.jpg'
          }
        ];
         }`

**Comment by naveen554** [comment]
@idomiralin , you need to install "npm install --save font-awesome angular-font-awesome" and do some changes in your angular-cli.json

https://www.npmjs.com/package/angular-font-awesome

**Comment by idomiralin** [comment]
@naveen554 thank you, but now I can see two small boxes instead of arrows. Do you have any idea why?

These are the steps I followed:

1. npm install --save font-awesome angular-font-awesome
2. Import the FontAwesomeModule in app.module.ts
3. I added in angular.json (I am using angular 8):
        "styles": [
             "src/styles.scss",
             "node_modules/font-awesome/scss/font-awesome.scss"
          ]

**Comment by marcosrbertuol** [comment]
any lucky ? Same problem here

**Comment by marcosrbertuol** [comment]
15 hours trying..... that's a shame, the gallery is great but I'm gonna have to move on from it

**Comment by naveen554** [comment]
@idomiralin  "node_modules/font-awesome/scss/font-awesome.scss" is this exact path ? i think "./node_modules/font-awesome/scss/font-awesome.scss" is the exact path and 2 places you need to add "node_modules/font-awesome/scss/font-awesome.scss"  this in angular.json file . pls check this once and actually i think  this is bug for updated angular versions >5

**Comment by gitalvininfo** [comment]
What I do is instead of pasting the font-awesome.css in angular.json I pasted it on my styles.scss like 
@import '~font-awesome/css/font-awesome.css'; and it works fine.

**Comment by gitalvininfo** [comment]
The '~' means it will tell angular to look for font-awesome inside the node_modules. In your case maybe your path is incorrect?

**Comment by naveen554** [comment]
@gitalvininfo is arrows are displaying   ?

**Comment by gitalvininfo** [comment]
Yes both arrows and close button displays but is there any solution that i
could set the width of the preview? Thanks for response.

**Comment by zigang93** [comment]
same issue here.. i am using angular 8
